test/Nancy.Authentication.Basic.Tests/project.json

```diff
@@ -17,13 +18,22 @@
 
     "frameworks": {
         "net452": {
-            "dependencies": {
-                "FakeItEasy": "2.0.0"
-            },
             "frameworkAssemblies": {
                 "System.Runtime": { "type": "build" },
                 "System.Threading.Tasks": { "type": "build" }
             }
+        },
+        "netcoreapp1.0": {
+            "buildOptions": {
+                "define": [ "CORE" ]
+            },
+            "dependencies": {
+                "Microsoft.NETCore.App": {
+                    "type": "platform",
+                    "version": "1.0.0"
+                }
+            },
+            "imports": [ "netstandard1.6" ]

```

Comment:
Huh. So I don't! I am sure I had to add it or something like it back over at FakeItEasy. Removed. Thanks. Also, we don't use the `"CORE"` definition, so I struck that while I'm at it.